webpack.common.config.ts

```diff
@@ -38,6 +38,9 @@ var defaultMeta = {
     'the community to create or try an exploration today!'
 };
 
+const GA_ANALYTICS_ID = '';
```

Comment:
Can you add comments above this to explain that they are populated during the deployment process?

Also, maybe explain the format -- I don't understand why you need JSON.stringify() below for a string. If this is not a string it may be worth renaming these vs the usages below to differentiate them?